src/pages/components/code.mdx

```diff
@@ -33,7 +33,8 @@ import ColorCode from '!!raw-loader!../../examples/typography/code/Color.tsx';
 
 ### Size
 
-Code comes in small, medium, and large. The default size is medium.
+Code comes in small, medium, and large. By default, size is inherited from
+the parent element.
```

Comment:
@sharondartist I realize this updated text uses passive voice, but it maps to copy constructs already in use:
- https://garden.zendesk.com/components/dots#size
- https://garden.zendesk.com/components/inline#color
- https://garden.zendesk.com/components/spinner#size